t/test.pl

```diff
@@ -1023,7 +1025,7 @@ sub _fresh_perl {
 
     # Use the first line of the program as a name if none was given
     unless( $name ) {
-        ($first_line, $name) = $prog =~ /^((.{1,50}).*)/;
+        my ($first_line, $name) = $prog =~ /^((.{1,50}).*)/;
```

Comment:
This doesn't look right, it shadows the outer $name instead of updating it.

Reply:
Probably should be `(my $first_line, $name) = ...`

Reply:
yep.